darkcluster/src/main/java/com/linkedin/darkcluster/impl/DarkClusterStrategyFactoryImpl.java

```diff
@@ -65,15 +67,15 @@
   private final Random _random;
   private final LoadBalancerClusterListener _clusterListener;
   private final DarkClusterVerifierManager _verifierManager;
-  private final ConstantQpsRateLimiter _rateLimiter;
+  private final ConstantQpsRateLimiterFactory _rateLimiterFactory;
 
   public DarkClusterStrategyFactoryImpl(@Nonnull Facilities facilities,
```

Comment:
Let's make this constructor the last constructor in the class

Reply:
Moved this constructor to the end